.github/workflows/zeebe-update-long-running-migrating-benchmark.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  workflow_dispatch: { }
+  release:
+    types: [published]
+
+jobs:
+  setup-mixed-benchmark:
+    name: Update Long Running Migrating Benchmark with mixed load
+    uses: ./.github/workflows/zeebe-benchmark.yml
+    secrets: inherit
+    with:
+      name: long-running-migrating-benchmark
+      cluster: zeebe-cluster
+      cluster-region: europe-west1-b
+      ref: main # we run the latest changes
```

Comment:
:x: That's not what we want though. This benchmark shouldn't run the latest code from main but always the newest patch release of the newest minor version. You need to query releases, determine the correct version and then use that version as the zeebe tag.